contracts/HenkakuBadge.sol

```diff
@@ -76,18 +87,25 @@ contract HenkakuBadge is ERC1155, Ownable {
         emit UpdateBadge(_tokenId, _mintable);
     }
 
-    function mint(uint256 _tokenId) public onlyExistBadge(_tokenId) {
+    function mint(uint256 _tokenId)
```

Comment:
If it is the original ERC1155, I need to change multiple pieces by receiving the amount, etc. Since it looks like ERC721 in this case, I think it might be OK to change it to ERC721.

Reply:
>  think it might be OK to change it to ERC721.
one of the goods of the badge is to supply the same Metadata with certain supply 
I cannot check opeansea with goerli but you will feel the sense of erc1155 once you see